site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5148,7 +5172,7 @@ public function getCourseForUserId($user_id, bool $archived = false, bool $dropp
 FROM courses_users u
 INNER JOIN courses c ON u.course=c.course AND u.term=c.term
 INNER JOIN terms t ON u.term=t.term_id
-WHERE u.user_id=? ${include_archived} AND ${force_nonnull} (u.registration_section IS NOT NULL OR u.user_group<>4)
+WHERE u.user_id=? ${include_archived} AND ${force_nonnull} (u.registration_section IS NOT NULL OR u.user_group<>5)
```

Comment:
What is the purpose of this change?

Reply:
This is a remnant of the first way I tried to do this, and it is obsolete now. I have reverted it. 40989cc